frontend/src/interfaces/SignupFormData.ts

```diff
@@ -0,0 +1,8 @@
+interface SignupFormData {
+  username: string;
```

Comment:
```suggestion
  userName: string;
```

mismatch between fields name in backend and frontend.

Reply:
I will change it in the backend